models/sipnet/R/#write_restart.SIPNET.R#

```diff
@@ -0,0 +1,126 @@
+#-------------------------------------------------------------------------------
```

Comment:
You can delete this file if you're not using it.

Reply:
Probably a backup, but yeah no need to keep around

Reply:
done